testsuite/features/step_definitions/command_steps.rb

```diff
@@ -5,6 +5,27 @@
 require 'timeout'
 require 'nokogiri'
 
+# Sanity checks
+
+Then(/^"([^"]*)" should have a FQDN$/) do |host|
+  node = get_target(host)
+  result, return_code = node.run('hostname -f')
```

Comment:
What about extending this a little? We can also run `python -c "import socket; socket.getfqdn()"` and compare its result with `hostname -f`. There were cases in the past, when those results differed ;-)

Reply:
As I told you, when I tried, that python socket was strictly equivalent to `hostname -f`, both in negative and positive cases. But I'm willing to believe you, so i'll add it.

Reply:
Done.